src/test/test_bitcoin.cpp

```diff
@@ -25,7 +25,7 @@
 
 CClientUIInterface uiInterface; // Declared but not defined in ui_interface.h
 CWallet* pwalletMain;
-libzerocash::ZerocashParams *pzerocashParams;
+ZCJoinSplit *pzcashParams;
```

Comment:
No it is not. That's how the tests have different initialization code than zcashd. :)